controllers/dashboardRoutes.js

```diff
@@ -0,0 +1,38 @@
+const { ProjectEmployee, Project, Task } = require('../models');
+
+const router = require('express').Router();
+
+router.get('/', async (req, res) => {
+    const projectEmployee = await ProjectEmployee.findAll({where: {employee_id: req.session.userId}});
+    const projects = [];
+    const tasks = await Task.findAll({where: {employee_id: req.session.userId}});
+    for (i in projectEmployee) {
```

Comment:
In a new line above this add the below code to get the plain data.
const tasks = await taskData.map((task) => task.get({ plain: true }));